riva/clients/nmt/riva_nmt_t2t_client.cc

```diff
@@ -90,23 +95,72 @@ translateBatch(
   }
 }
 
-int countWords(const std::string& text) {
-    int wordCount = 0;
-    bool wasSpace = true; 
-    for (char c : text) {
-        if (std::isspace(c)) {
-            if (!wasSpace) {
-                wordCount++;
-            }
-            wasSpace = true;
-        } else {
-            wasSpace = false;
-        }
-    }
-    if (!wasSpace) {
+int
+countWords(const std::string& text)
+{
+  int wordCount = 0;
+  bool wasSpace = true;
+  for (char c : text) {
+    if (std::isspace(c)) {
+      if (!wasSpace) {
         wordCount++;
+      }
+      wasSpace = true;
+    } else {
+      wasSpace = false;
     }
-    return wordCount;
+  }
+  if (!wasSpace) {
+    wordCount++;
+  }
+  return wordCount;
+}
+
+std::string
+ReadUserDictionaryFile(const std::string& dictionary_file)
```

Comment:
can we name the function better? since it is dnt phrases file, not user dictionary